spec/integ/crypto/megolm-backup.spec.ts

```diff
@@ -540,6 +551,85 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("megolm-keys backup (%s)", (backe
             expect(result.imported).toStrictEqual(expectedTotal - decryptionFailureCount);
         });
 
+        it("Should report failures when decryption works but import fails", async function () {
+            // @ts-ignore - mock a private method for testing purpose
+            aliceCrypto.importBackedUpRoomKeys = jest
```

Comment:
Again: by mocking out `importBackedUpRoomKeys`, these are unit tests rather than integration tests. We have no real evidence that the actual `RustCrypto` implementation behaves the same way that these mocks do.

Could we not make up some data which causes `RustCrypto.importBackedUpRoomKeys` to report a failure, without having to mock this out?